core/templates/dev/head/pages/state_editor/state_content_editor_directive.html

```diff
@@ -37,4 +50,42 @@
     <button type="button" class="btn btn-default pull-right" ng-click="cancelEdit()">Cancel</button>
     <div style="clear: both;"></div>
   </div>
+  <div ng-if="isCardHeightLimitReached && showCardHeightLimitWarning">
+    <span class="oppia-card-height-limit-warning">
+      <i class="material-icons" uib-tooltip="Hide warning" ng-click="hideCardHeightLimitWarning()">&#xE88F;</i>
+      <span translate="I18N_CARD_HEIGHT_LIMIT_WARNING_MESSAGE"></span>
+    </span>
+  </div>
 </div>
+
+<style>
+  .oppia-card-height-limit-warning {
+      color:#0277BD;
+      padding-top: 2px;
+  }
+  .shadow-preview-card {
+    left: -30000px;
+    max-width: 560px;
+    position: fixed;
+    top: -30000px;
+  }
+  .rte-viewer > p {
```

Comment:
Are we expecting to have this comment on the top?
https://github.com/oppia/oppia/blob/04f4b8f26b45b7b7431c1d48a29dbdd732c45fd9/core/templates/dev/head/pages/exploration_player/conversation_skin_embed_directive.html#L388-L395

I'm not sure whether `conversation_skin_embed_directive.html` is in sync with `oppia.css` style as mentioned in the comment? @bansalnitish, can you please check? Also, is there any better way to do this as now we have this css in lots of other places?

Reply:
Would it be fine to move the css for `rte-viewer` to `oppia.css`?

Reply:
@DubeySandeep, yes these both should be in sync with each other. Also, the value (of line-height) in the link attached by you is different than what we currently have. (Refer - https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html#L249)

Reply:
> yes these both should be in sync with each other.

I think this change was expected from #5981, would you like to make a PR for this? (Let me know if you want me to do it, I'll be happy to do it. :))

Also, can we have an answer to this:
> is there any better way to do this (adding comment and keep those style in sync in multiple places) as now we have this css in lots of other places?

Reply:
One option can be to add all the css which needs to be in sync in `oppia.css` only or if that is not feasible we can add a test which ensures that the css remains in sync. What do you think @DubeySandeep @bansalnitish?

Reply:
@bansalnitish, can you please please check whether the options suggested by @ankita240796 in the above comment will be fine?

Reply:
@ankita240796, any update on this?

Reply:
Hi @DubeySandeep, should I issue a separate PR for this moving the common css to `oppia.css` or making the changes here will be fine?

Reply:
I feel it would be better to do it in a separate PR after opening an issue for the same and discussing a bit further on that. What do you suggest?

Reply:
Sounds good! (Creating a separate thread discussing the CSS syn issue.)

For now can you please add a similar comment at the top and create an issue for the same with a link to all places.

Reply:
@ankita240796, I hope you've gone through this comment, do let me know if it's not clear. Thanks! :)